### DISCUSSION TITLE
idea - compact mode for main windows

### DISCUSSION THREAD

**Discussion by Sontarr** [seerge/g-helper#2286] [Ideas]
Thanks for your great work with G-Helper, the active development and continued support. I think it's an essential software for Asus laptop users.

But with more and more features added, the main window tends to get a little crowded with features that are not used daily. I don't often change the options for the AniMe matrix or the keyboard RGB. And while I see there may be people who are using the new visual mode options extensivly, I don't use them ever.

What do you think about a button in the title bar for example, to switch between a standard and a compact mode for the main window?

**Comment by seerge** [comment]
@Sontarr hello,

You already have a super-compact version of that a right-click context menu when you click on tray icon :)

I don't want to overcomplicate UI and do different UI versions as it will be a hell to support
